src/redux/param/coline-action.ts

```diff
@@ -0,0 +1,104 @@
+import { AppDispatch, RootState } from '../index';
+import { InterchangeInfo } from '../../constants/constants';
+import { setColineBulk } from './action';
+import { setSelectedColine } from '../app/action';
+
```

Comment:
The first 3 methods should be utils for the side panel. Better to move them to the relevant folder as this folder is solely for Redux. (Keep things clean). 

Also the coline-related actions from `actions.ts` could be moved to here?